tests/should-pass/iati-activities/iati-activity/result/reference/reference-element-present-with-namespace-element.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx" xmlns:example-namespace="http://example.org/example-namespace/ns#"><!-- Include namespace declaration. -->
+  <iati-activity>
+    <iati-identifier></iati-identifier>
+    <reporting-org type="xx" ref="xx">
+      <narrative>Organisation name</narrative>
+    </reporting-org>
+    <title>
+      <narrative>Xxxxxxx</narrative>
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative>
+    </description>
+    <participating-org role="xx"></participating-org>
+    <activity-status code="xx"/>
+    <activity-date type="xx" iso-date="2013-11-27"/>
+    <activity-date type="xx" iso-date="2013-11-27">
+      <narrative>Some stuff ere</narrative>
+    </activity-date>
+    <result type="xx" aggregation-status="0">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <reference vocabulary="xx" code="xx"><!-- PASS: Optional `reference` element present, with two mandatory attributes. -->
+        <example-namespace:namespace-element></example-namespace:namespace-element><!-- Optional namespace element included. -->
```

Comment:
Could be clearer that this is what is being tested.

Reply:
I've made changes in 59a684c - is it clearer now?